tests/e2e/smoke/ocr2_proxy_test.go

```diff
@@ -0,0 +1,57 @@
+package smoke_test
+
+import (
+	"time"
+
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+	"github.com/smartcontractkit/chainlink-terra/tests/e2e"
+	"github.com/smartcontractkit/chainlink-terra/tests/e2e/common"
+	tc "github.com/smartcontractkit/chainlink-terra/tests/e2e/smoke/common"
+	"github.com/smartcontractkit/integrations-framework/actions"
+)
+
+var _ = Describe("Terra OCRv2 Proxy @ocr_proxy", func() {
+	var state *tc.OCRv2State
+
+	BeforeEach(func() {
+		state = &tc.OCRv2State{}
+		By("Deoloying the cluster", func() {
```

Comment:
```suggestion
		By("Deploying the cluster", func() {
```

Reply:
Good catch! I copy pasted from another test and it had the issue so I fixed it in all the copy pasted areas and found some others.